apps/web/src/components/organization-select.tsx

```diff
@@ -82,55 +96,81 @@ export default function OrganizationSelect() {
     mutationFn: async (insertData) => createOrganization(insertData),
     onSuccess: (data) => {
       if (!!data) {
+        toast({
+          title: "Success",
+          description: "Organization created successfully",
+        });
         router.push(`/dashboard/${data.name}/overview`);
         setIsModalOpen(false);
         refetch();
       }
     },
+    onError: (error) => {
+      toast({
+        title: "Error",
+        description: error.message,
+      });
+    }
   });
 
-  const onSubmit: SubmitHandler<OrganizationForm> = (data) => {
+  const onSubmit: SubmitHandler<InsertOrganization> = (data) => {
     createOrganizationMutation({ name: data.name });
   };
 
-  const onError: SubmitErrorHandler<OrganizationForm> = (error) => {
+  const onError: SubmitErrorHandler<InsertOrganization> = (error) => {
     console.error(error);
   };
 
-  return (
+  return !isLoading && (
     <>
-      <Select
-        value={organizationFromPathname}
-        onValueChange={(newValue) => {
-          selectOrganization(newValue);
-        }}
-      >
-        <SelectTrigger className="w-[180px]">
-          <SelectValue placeholder={organizationFromPathname} />
-        </SelectTrigger>
-
-        <SelectContent>
-          <SelectGroup>
-            {organizations?.map((org) => (
-              <SelectItem
-                value={org.name}
-                key={org.id}
-                className="hover:cursor-pointer"
-              >
-                {org.name}
-              </SelectItem>
-            ))}
-            <div
-              className="text-slate relative m-auto flex w-full cursor-default select-none items-center rounded-sm py-1.5 pl-8 pr-2 text-sm font-light text-slate-400 outline-none hover:cursor-pointer focus:bg-accent focus:text-accent-foreground data-[disabled]:pointer-events-none data-[disabled]:opacity-50"
+      {
+        organizations && organizations.length > 0 ? (
```

Comment:
This could be change to:
```suggestion
        organizations?.length > 0 ? (
```

Reply:
Also aren't we requiring that user has to have at least one org?

Reply:
User now can have no orgs when he gets kicked of his last one.
`organizations && organizations.length > 0 ? (` Has to stay like this since org can be : `organizations: {
    id: number;
    name: string;
}[] | null | undefined`